examples/main.rs

```diff
@@ -28,6 +28,11 @@ fn main() {
                                     "event label",
                                     999999);
 
+   metrics_controller.record_floating_point_event("event category",
+                                                  "event action",
+                                                  "event label",
+                                                  999999);
```

Comment:
It would be better if the value here was a floating point value. For example, 99999.9